chapters/10-community-involvement.Rmd

```diff
@@ -9,7 +9,28 @@
 - NEWS file
 - reprex
 
+## Outline of action steps
 
+This chapter is very description-heavy with minimal code.
+
+- Creating a welcoming community with welcoming materials
+<!-- I grouped reprex and NEWS with the README because they seem related to the general welcoming presentation of a project, but they could go elsewhere. -->
+  - Text: the purpose of a README and what goes into one ([Mozilla reference](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), [R packages reference](https://r-pkgs.org/release.html#readme)).
+  - Text: making your project easy to use and understand with reproducible examples. `reprex`: not just for bug reports!
+  - Code: use the `reprex` package to create a minimal reproducible example
+  - Text: using a NEWS file ([reference](https://r-pkgs.org/release.html#news))
+  - Code: `usethis::use_news_md()`
+- Code of Conduct
+  - Text: inclusivity, safety, the purpose of a Code of Conduct and what goes into one ([reference](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/)).
+  - Code: `usethis::use_code_of_conduct()`
+- Contributing guidelines
+  - Text: the purpose of Contributing guidelines and what they can contain ([reference](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/)). Emphasize that the different sections are quite project-specific (i.e. your project might not use communication channels outside of GitHub).
+  - Code: `usethis::use_tidy_contributing()`
+- Licensing
```

Comment:
I think we should move licensing up to above contributing guidelines; licenses are for folks to know how to use AND contribute to software, so then the flow of the chapter would continue to narrow down in scope.